microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -151,14 +149,14 @@ Red Hat Customer Portal users can enable errata notifications in the account set
 Red Hat Customer Portal user accounts must have systems registered and consuming {microshift-short} entitlements for {microshift-short} errata notification emails to generate.
 ====
 
-This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {microshift-short} {product-version}. Versioned asynchronous releases, for example with the form {microshift-short} {product-version}.z, will be detailed in the following subsections.
+This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {microshift-short} {product-version}. Versioned asynchronous releases, for example with the form {microshift-short} {product-version}.z, are detailed in the following subsections.
 
-//TODO uncomment and verify info prior to merge freeze
-//[id="microshift-4-16-0-dp"]
-//=== RHEA-2024:0043 - {microshift-short} 4.16.0 bug fix and security update advisory
+//TODO verify info prior to merge freeze
+[id="microshift-4-16-0-dp"]
+=== RHEA-2024:0043 - {microshift-short} 4.16.0 bug fix and security update advisory
 
-//Issued: 2024-04-03
+Issued: 2024-06-03
```

Comment:
Is this the anticipated release date for MicroShift 4.16? I'm unfamiliar - do we typically add future release dates?

Reply:
I would, personally :) just in case the date moves, so folks don't have an expectation.